src/libraries/System.ComponentModel.Composition/ref/System.ComponentModel.Composition.cs

```diff
@@ -174,9 +174,9 @@ public enum ImportSource
     public partial class InheritedExportAttribute : System.ComponentModel.Composition.ExportAttribute
     {
         public InheritedExportAttribute() { }
-        public InheritedExportAttribute(string contractName) { }
-        public InheritedExportAttribute(string contractName, System.Type contractType) { }
-        public InheritedExportAttribute(System.Type contractType) { }
+        public InheritedExportAttribute(string? contractName) { }
+        public InheritedExportAttribute(string? contractName, System.Type? contractType) { }
```

Comment:
Ditto.  On all such cases.  I'll stop commenting on them.

Reply:
This one has documented to accept nulls